openmls/src/group/core_group/new_from_external_init.rs

```diff
@@ -29,28 +25,29 @@ impl CoreGroup {
         backend: &impl OpenMlsCryptoProvider,
         params: CreateCommitParams,
         tree_option: Option<&[Option<Node>]>,
-        verifiable_public_group_state: VerifiablePublicGroupState,
+        group_info: GroupInfo,
+        external_pub: &[u8],
+        interim_transcript_hash: &[u8],
     ) -> Result<ExternalCommitResult, ExternalCommitError> {
-        let ciphersuite = verifiable_public_group_state.ciphersuite();
-        if verifiable_public_group_state.version() != ProtocolVersion::Mls10 {
-            return Err(ExternalCommitError::UnsupportedMlsVersion);
-        }
+        let ciphersuite = group_info.group_context().ciphersuite();
+        // TODO: Check version
```

Comment:
Assign issue?

Reply:
Ah, right. I thought we will introduce the version later here. But this is not the case and I confused it with another issue. As we already have it in the group context, I can check here :+1: